pennylane/qaoa/layers.py

```diff
@@ -72,7 +72,7 @@ def cost_layer(gamma, hamiltonian):
 
         .. code-block:: python
 
-            dev = qml.device('default.qubit', wires=2)
+            dev = qml.device('default.mixed', wires=2)
```

Comment:
`DefaultQubit` allows the execution of any operator that has a matrix defined, thus `Evolution` would NOT be expanded (because it has a `matrix`). 

Using `DefaultMixed` we ensure that `Evolution` is expanded when running `qml.draw(circuit, expansion_strategy="device")(0.5)`

Reply:
Thanks for letting me know :)